examples/cis3-nft-sponsored-txs/src/lib.rs

```diff
@@ -28,35 +28,32 @@
 //! submits the transaction on behalf of the user and pays the transaction fee
 //! to execute the transaction on-chain.
 //!
-//! Concordium smart contracts currently have no way to query the corresponding
-//! public key(s) of an account within the smart contract code. For
-//! the time being this smart contract uses a `public_key_registry` that
-//! allows only the owner of the contract instance (or the account itself) to
-//! register a public key for a given account. Once an account has a public key
-//! registered, the mapping between the public key and the account can not be
-//! updated anymore. The users/clients are advised to check if the correct
-//! public key is registered to their account or register a public key
-//! themselves (if they don't want to make use of the sponsored transaction
-//! feature of this smart contract). This smart contract assumes a
-//! trusted third-party service provider since the use case is for users/clients
-//! that pay fiat money off-chain to the third-party service to request the
-//! submission of sponsored transactions on their behalf.
-//!
-//! The Concordium team is working on exposing the public key within the smart
-//! contract code and this feature is planned to be included in the next
-//! protocol update in 6-12 months. After that protocol update, the
-//! `public_key_registry` will not be necessary anymore.
-//!
 //! As follows from the CIS2 specification, the contract has a `transfer`
 //! function for transferring an amount of a specific token type from one
 //! address to another address. An address can enable and disable one or more
 //! addresses as operators with the `updateOperator` function. An operator of
-//! some address is allowed to transfer any tokens owned by this address.
+//! an address is allowed to transfer any tokens owned by this address.
 //! As follows from the CIS3 specification, the contract has a `permit`
-//! function. It is the sponsored counterparts to the `transfer/updateOperator`
+//! function. It is the sponsored counterpart to the `transfer/updateOperator`
 //! function and can be executed by anyone on behalf of an account given a
-//! signed message by the private key corresponding to the public key that
-//! is registered for that account.
+//! signed message.
+//!
+//! Concordium supports natively multi-sig accounts. Each account address on
+//! Concordium is controlled by one (or several) credentials (real-world
```

Comment:
Reading this sentence skipping `(or several)` is off, so I suggest either:
```suggestion
//! Concordium is controlled by one or several credentials (real-world
```
or
```suggestion
//! Concordium is controlled by one (or several) credential(s) (real-world
```